routes/chatRoutes.js

```diff
@@ -0,0 +1,14 @@
+const express=require('express');
+const router=express.Router();
+import {getChatRooms, getChatRoomsByTopic, AddChatRoom } from '../controllers/roomController';
+import {getUsers, getUsersById, setUser} from '../controllers/userController';
+
+router.route('/chatrooms').get(getChatRooms).post(AddChatRoom);
+router.route('/chatrooms/:topic').get(getChatRoomsByTopic);
```

Comment:
@YousefAsilah12 in the DB, is the topic an id or string?  if its mongo id then I think that
@FaridBash should make the route be /:topic_id
for example: "/chatrooms/1234-5332-9865"